code/modules/unit_tests/monkey_business.dm

```diff
@@ -0,0 +1,59 @@
+/**
+ * Monkey Business
+ *
+ * This unit test spawns a predefined number of monkies, each of which
+ * are set to have a 100% chance of attempting to use something next to them each Life
+ *
+ * This test basically just checks to see if attack procs are working correctly,
+ * but its also hilarious and fun to watch locally.
+ */
+/datum/unit_test/monkey_business
+	priority = PRIORITY_LOW + 1 // should be the last test to run
+	var/monkey_count = 50
+	var/monkey_timer = 30 SECONDS
+	var/start_runtimes = 0
+	var/list/monkey_list
+	var/list/running = TRUE
```

Comment:
This seems like a simple boolean, why is it a list variable?

Reply:
Because I'm stupid